backend/openapi.yaml

```diff
@@ -3778,6 +3790,18 @@ components:
           type: number
           minimum: 0
           example: 1802.88
+        additional_work_during_construction:
+          description: Amount of apartment's additional work done during construction
+          type: number
+          nullable: true
+          minimum: 0
+          example: 0.0
+        index_adjusted_additional_work_during_construction:
+          description: Amount of apartment's additional work done during construction, after index adjustment
+          type: number
+          nullable: true
```

Comment:
Check if this is nullable, not maked so in the dataclass.

Reply:
Good catch, fixed :)